proto/gen/einride/example/freight/v1/aiptest.pb.go

```diff
@@ -0,0 +1,17 @@
+// Code generated by protoc-gen-go-aip-test. DO NOT EDIT.
+
+package examplefreightv1
+
+import (
+	testing "testing"
+)
+
+// ServicesTestSuiteConfigProviders embeds providers for all services.
+type ServicesTestSuiteConfigProviders interface {
```

Comment:
Given that we expect this to be the main entrypoint for writing these tests, is it worth it to have a less verbose name? I'm thinking something along the lines of `[package]TestProvider` in this case (`FreightTestProvider`).

Reply:
Greate idea! Naming is hard! I will update